src/index.ts

```diff
@@ -134,6 +149,29 @@ export class CheckoutInstance {
     }
   }
 
+  private mapOptionsToManifest(manifest: CheckoutManifest, options: CheckoutOptions) {
```

Comment:
I think this function is a bit confusing because it destructively changes (in-place) the existing manifest **and** it returns the same manifest, making it seem it's making a changed copy. I think it should either make a fresh manifest and return it or change the existing one without returning it.